packages/scripts/src/ManualRelay.js

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+// This script allows you to manually relay a particular across deposit.
+// Example:
```

Comment:
So this manually relays a deposit originating from chain ID 10 on chain ID 1? Maybe its worth a quick note explaining why you specify _1 and _10 NODE_URL here

Reply:
Sure

Reply:
Done